dali/operators/optical_flow/turing_of/optical_flow_turing.cc

```diff
@@ -85,6 +85,8 @@ OpticalFlowTuring::~OpticalFlowTuring() {
     hintsbuf_.reset(nullptr);
   }
   auto err = turing_of_.nvOFDestroy(of_handle_);
+  // unload lib no matter if it was successful
+  UnloadTuringOpticalFlow();
```

Comment:
how about having a RAII object that does dlclose on destructor? A unique_ptr would do

Reply:
Done